packages/env/lib/build-docker-compose-config.js

```diff
@@ -142,11 +142,24 @@ module.exports = function buildDockerComposeConfig( config ) {
 				image: 'composer',
 				volumes: [ `${ config.configDirectoryPath }:/app` ],
 			},
+			phpunit: {
+				image: 'wordpressdevelop/phpunit',
+				depends_on: [ 'wordpress' ],
```

Comment:
```suggestion
				depends_on: [ 'tests-wordpress' ],
```

Same for `tests-cli`.

Reply:
great idea